src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
It looks like you're linking an SCSS file directly in your HTML. Browsers can't understand SCSS files, so you need to link the compiled CSS file instead. Make sure to compile your SCSS to CSS and then link to the CSS file.